patches/base-path.diff

```diff
@@ -180,15 +180,15 @@ Index: code-server/lib/vscode/src/vs/server/node/webClientServer.ts
  		const data = (await util.promisify(fs.readFile)(filePath)).toString()
  			.replace('{{WORKBENCH_WEB_CONFIGURATION}}', escapeAttribute(JSON.stringify({
  				remoteAuthority,
-@@ -279,6 +284,7 @@ export class WebClientServer {
- 				developmentOptions: { enableSmokeTestDriver: this._environmentService.driverHandle === 'web' ? true : undefined },
- 				settingsSyncOptions: !this._environmentService.isBuilt && this._environmentService.args['enable-sync'] ? { enabled: true } : undefined,
+@@ -285,6 +290,7 @@ export class WebClientServer {
+ 				folderUri: resolveWorkspaceURI(this._environmentService.args['default-folder']),
+ 				workspaceUri: resolveWorkspaceURI(this._environmentService.args['default-workspace']),
  				productConfiguration: <Partial<IProductConfiguration>>{
 +					rootEndpoint: base,
```

Comment:
I didn't change anything here. I assume this is fine

Reply:
Yup looks good to me.